showcase/angular.json

```diff
@@ -19,7 +19,7 @@
             "assets": [
               {
                 "glob": "mdi.svg",
-                "input": "../node_modules/@mdi/angular-material",
+                "input": "../node_modules/@nationalbankbelgium/stark-ui/node_modules/@mdi/angular-material",
```

Comment:
I think we need to review this. This change is caused by the change in the tsconfig.json to link the stark-ui in the showcase.

Are we sure this will work in a production build? Otherwise we should revert this change as well as in the tsconfig.json and just keep them in our own fork only in development.